dom.js

```diff
@@ -0,0 +1,48 @@
+const form = document.querySelector('.form');
+
+const render = () => {
+  const booksctn = document.querySelector('#books-ctn');
+  booksctn.innerHTML = '';
+
+  for (let i = 0; i < myLibrary.length; i++) {
+    booksctn.innerHTML += ` 
+    <div class= "container col-4 mb-3">
+      <div class="card card-sh mt-2 bg-frm" >
+        <div class="card-body" data-id=${i}>
+          <h5 class="card-title"> ${myLibrary[i].title} </h5>
+          <p class="card-text"> ${myLibrary[i].title} is written by ${myLibrary[i].author} and it has ${myLibrary[i].pages} pages </p>
+          <p> book is ${myLibrary[i].read ? ' read' : 'unread'} </p>
+          
+          <a href="#" class=" read "> change read status </a>
```

Comment:
- [ ] Use a button tag instead of a link. Javascript actions should be triggered by buttons while links take care of navigation.
[Read more here.](http://web-accessibility.carnegiemuseums.org/content/buttons/)